### PR TITLE
doc: update plugin version in the readme

### PR DESCRIPTION
**Pull request by humblec**
This commit update the readme with the driver version available
in the deploy directory along with some other markdown corrections
in the same file.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

> /kind documentation

-->
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,20 @@
...
+
 ```
 $ sudo ./bin/iscsiplugin --endpoint tcp://127.0.0.1:10000 --nodeid CSINode
 ```
 
 #### Get plugin info
+
 ```
 $ csc identity plugin-info --endpoint tcp://127.0.0.1:10000
-"ISCSI"	"0.1.0"
```
Comment (andyzhangx) [inline README.md:13]: first alpha version should be `0.1.0` and when `1.0.0` should be GA version

Comment (humblec) [reply README.md:13]: @andyzhangx the driver version in the code is `1.0.0`, https://github.com/kubernetes-csi/csi-driver-iscsi/blob/master/pkg/iscsi/driver.go#L44 , so this correction.  Considering the plan is to release I think we have to confirm the release state and then correct accordingly. Looks like its good to release `alpha` first . If we are in agreement, I will change the driver version in the code then? wdyt?

Comment (andyzhangx) [reply README.md:13]: let's use `0.1.0` for alpha first, thanks.

Comment (humblec) [reply README.md:13]: @andyzhangx done. ptal . thanks

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**